tests/test_parent_ci.py

```
@@ -0,0 +1,86 @@
+    assert "make test WITH_ACT=1" not in ci_workflow, (
+        "expected parent CI to leave act validation to a separate workflow"
+    )
+    assert "permissions:\n  contents: read" in act_workflow, (
+        "expected parent act-validation workflow to restrict GITHUB_TOKEN"
+    )
+    assert "ACT_VERSION:" in act_workflow, (
+        "expected parent act-validation workflow to declare an act version"
+    )
+    assert (
+        "act_Linux_x86_64.tar.gz" in act_workflow and "${ACT_VERSION}" in act_workflow
+    ), "expected parent act-validation workflow to build the act URL from ACT_VERSION"
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

(PT018)